Contoso.Mathlib/Roundings.cs

```diff
@@ -14,6 +14,10 @@ public static int Ceiling(double d)
             return (int)(d + 1);
         }
 
+        public static void Dummy()
+        {
+            //TODO: Add body
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1135)